docs/dictionaries.md

```diff
@@ -55,19 +56,25 @@ Term from the Neurobagel vocabulary.
     multiple subject ID columns for situations where a study is using more than
     one ID scheme.
 
+!!! note
+    The `Identifies` annotation key is required for columns containing unique observation identifiers 
```

Comment:
Like the note! But isn't the real reason we have `Identifies` for purely modeling and validation reasons? I.e. so we have a unique attribute for a column that is about Participants because if we didn't add the `"Identifies"`, every possible column would be valid as long as it has an `"IsAbout"` section?

```suggestion
    The `Identifies` annotation key is currently required to validate annotations for columns about unique 
    observation identifiers  (e.g., participant or session IDs). The `"Identifies"` key should only be used for these
    columns and its value should be an informative string value describing the type/level of observation 
    identified. This required key is currently only used for validation and its value will not be processed by
    Neurobagel.
```